defaults/main.yml

```diff
@@ -117,3 +118,8 @@ ovos_tts_plugin_piper: ""
 
 # TTS - ovos-tts-plugin-server
 ovos_tts_plugin_server: "http://mycroft.blue-systems.com:59125"
```

Comment:
I do not believe this url works for `ovos-tts-plugin-server`  That is a url for `ovos-tts-plugin-mimic3`

there are several others available for `plugin-server`

- `https://tts.smartgic.io/mimic/stt`
- `https://tts.smartgic.io/mimic3/stt`
- `https://mimic3.ziggyai.online.stt`
- `https://tts.smartgic.io/piper/stt`
- `https://pipertts.ziggyai.online/stt`

Reply:
they all finish in stt ?

Reply:
for TTS, public servers should be consistent in voices they support, we recommend we stick to piper

Related https://github.com/OpenVoiceOS/ovos-tts-server-plugin/pull/5

as long as the default voice is alan pope server side (and not mimic1) we can add more urls above, but for now that means servers should run piper essentially

Reply:
this was merged, so there should be no need for a "host" at all.  It can use the default public ones that are provided with the plugin